backend/tests/auth/test_auth_service.py

```
@@ -0,0 +1,660 @@
+import pytest
+from unittest.mock import AsyncMock, patch, MagicMock
+from fastapi import HTTPException, status
+from bson import ObjectId
+from jose import JWTError
+from firebase_admin import auth as firebase_auth
+from datetime import datetime, timedelta, timezone
+from app.auth.service import AuthService
+from app.auth.security import get_password_hash, create_refresh_token, verify_password
```

Ruff (0.12.2)

`app.auth.security.get_password_hash` imported but unused

Remove unused import

(F401)